File: Funcional1.java
public class Funcional1 {
    public static String func(String nombre) {
        return "Mi nombre es " + nombre; // Concatenación de cadenas
    }

    public static void main(String[] args) {
        String mensaje = func("Darwin Cardozo");

        System.out.println(mensaje);
    }
}

File: Funcional1.cpp
#include <iostream>
#include <string>

int main() 
{
    std::string nombre = "Darwin Cardozo"; // Agrega el punto y coma al final
    std::string mensaje = "Hola, mi nombre es " + nombre; // Usa + para concatenar

    std::cout << mensaje << std::endl; // Imprime el mensaje
    return 0;
}